content/doc/book/installing/linux.adoc

```diff
@@ -147,7 +139,7 @@ sudo wget -O /etc/yum.repos.d/jenkins.repo \
 sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
 sudo dnf upgrade
 # Add required dependencies for the jenkins package
-sudo dnf install chkconfig java-devel
+sudo dnf install fontconfig java-11-openjdk
 sudo dnf install jenkins
```

Comment:
Does this need to be split over 2 `dnf install` lines?

Reply:
Not sure offhand.

Reply:
don't think so but pre-existing issue 😄